### PR TITLE
Review

### PR DESCRIPTION
**Pull request by jsnjack**

### CODE REVIEW

File: lxgettext/lxgettext.py [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+import os
+import re
+
+import polib
+
+COLOUR_GREEN = '\033[92m'
+COLOUR_END = '\033[0m'
+
+KEYWORD = "gettext"
+gettext_re = re.compile("""%s\(['"](.+?)['"]\)""" % KEYWORD)
```
Comment (muodov) [inline lxgettext/lxgettext.py:12]: this won't catch `gettext("hi there :') hola if ya hear!")`
Not sure it's a problem though

Comment (jsnjack) [reply lxgettext/lxgettext.py:12]: true, that is probably why they use a real parsers! ;) we just need to be aware of it

Comment (ema-fox) [reply lxgettext/lxgettext.py:12]: I don't like this brittleness.

Comment (jsnjack) [reply lxgettext/lxgettext.py:12]: @ema-fox me neither, but it is the simplest solution that does not require real parsing (and that is the problem as there is no all-in-one html, js, vue parser)

**Pull request closed without merging**